sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryAsyncClient.java

```diff
@@ -290,8 +290,8 @@ Mono<Response<SchemaRegistrySchema>> getSchemaWithResponse(String groupName, Str
                 context)
             .onErrorMap(ErrorException.class, SchemaRegistryAsyncClient::remapError)
             .handle((response, sink) -> {
-                final InputStream schemaInputStream = response.getValue();
-                final SchemaProperties schemaObject = SchemaRegistryHelper.getSchemaProperties(response);
+                final InputStream schemaInputStream = response.getValue().toStream();
```

Comment:
Why do we use `InputStream` in async client? Now that we have `BinaryData`, can we use `toFlux()` instead?

Reply:
InputStream was being used before, so didn't intend to change the logic as part of sync stack PR.
I've updated to use Flux<ByteBuffer> here, do verify if it looks good.